app/controllers/items_controller.rb

```diff
@@ -46,20 +46,35 @@ def update
     end
   end
 
-  # DELETE /items/1 or /items/1.json
-  def destroy
-    @item.destroy
-    respond_to do |format|
-      format.html { redirect_to items_url, notice: "Item was successfully destroyed." }
-      format.json { head :no_content }
+  def status_change
```

Comment:
In future maybe try to have this called in the #update so that our routes are restful.